src/main/java/randoop/test/RegressionCaptureGenerator.java

```diff
@@ -148,8 +149,22 @@ public RegressionChecks generateTestChecks(ExecutableSequence eseq) {
               continue;
             }
             // System.out.printf("Adding objectcheck %s to seq %08X%n", poc, s.seq_id());
-            PrimValue.EqualityMode equalityMode =
-                var.getType().isPrimitive() ? EQUALSEQUALS : EQUALSMETHOD;
+            PrimValue.EqualityMode equalityMode;
+
+            // GRT Fuzzing option's short fuzzing strategy results in ambiguous assertions
+            // for primitive numbers. Use == for primitives since Java automatically
```

Comment:
By "short", do you mean the `short` numeric type?  If so, please clarify, say, by putting `` ` `` around "short".